crates/sequencer/src/sequencer.rs

```diff
@@ -595,10 +600,24 @@ where
         )
     }
 
+    fn update_rpc_storage(&mut self, l2_height: u64) -> Result<(), anyhow::Error> {
+        let new_rpc_storage = self
+            .storage_manager
+            .create_storage_on_l2_height(l2_height)?;
+        // `send_replace` is superior to `send` for our use case. It never fails
+        // because it doesn't need to notify all receivers, unlike `send`, which
+        // we don't need. It will also keep working even if there are no
+        // receivers currently alive, which makes it easier to reason about the
+        // code.
```

Comment:
- What is "our use case" here? 
- "unlike `send` which we don't need" - Sentence not needed when we clarify what the use case is.

Reply:
Our case means that `send` will fail if a recv part is dead - that recv part is rpc module.

Reply:
Sure, i would add that to the comment replacing "our use case"